CHANGELOG.md

```diff
@@ -0,0 +1,35 @@
+## 2.0.0-SNAPSHOT
+
+- Adds support for new reference types in FaunaDB API 2.1
+  - `DatabaseV`, `ClassV`, `IndexV`, `FunctionV` and `KeyV`
+- Removes `Ref(string id)` function
```

Comment:
We need to say that references will have to be created with `new RefV` from now on.